src/extensions/figlet.py

```diff
@@ -0,0 +1,62 @@
+import arc
+import hikari
+
+
+from pyfiglet import Figlet
+
+plugin = arc.GatewayPlugin(name="figlet")
+
+style_choices = [
```

Comment:
I think it would be cleaner to have this as a dictionary and then build it into a CommandChoice list below when passing it to the option params